controllers/tests/controller_server/host_definer/common.py

```diff
@@ -0,0 +1,73 @@
+import unittest
+import logging
+from mock import patch
+from kubernetes.client.rest import ApiException
+
+from controllers.common.csi_logger import get_stdout_logger
+import controllers.tests.controller_server.host_definer.utils.utils as utils
+from controllers.servers.host_definer.watcher.csi_node_watcher import CsiNodeWatcher
+from controllers.servers.host_definer.watcher.host_definition_watcher import HostDefinitionWatcher
+from controllers.servers.host_definer.watcher.node_watcher import NodeWatcher
+from controllers.servers.host_definer.watcher.secret_watcher import SecretWatcher
+from controllers.servers.host_definer.watcher.storage_class_watcher import StorageClassWatcher
+import controllers.tests.controller_server.host_definer.settings as settings
+
+
+class MockLogHandler(logging.Handler):
+    records = []
+
+    def emit(self, record):
+        self.records.append(record.getMessage())
+
+
+class BaseSetUp(unittest.TestCase):
+    @classmethod
+    def setUpClass(cls):
+        logger = get_stdout_logger()
+        cls._mock_logger = MockLogHandler(level='DEBUG')
+        logger.addHandler(cls._mock_logger)
+
+    def setUp(self):
+        utils.patch_kubernetes_manager_init()
+        self.csi_node_watcher = utils.get_class_mock(CsiNodeWatcher)
+        self.host_definition_watcher = utils.get_class_mock(HostDefinitionWatcher)
+        self.node_watcher = utils.get_class_mock(NodeWatcher)
+        self.secret_watcher = utils.get_class_mock(SecretWatcher)
+        self.storage_class_watcher = utils.get_class_mock(StorageClassWatcher)
+
+        self._mock_logger.records = []
```

Comment:
IMO logger testing is not indicator for correct logic of the code so I think we need better ways to test parts that do "nothing" except logging when error